test/parallel/test-stream2-readable-wrap.js

```diff
@@ -77,14 +73,10 @@ function runTest(highWaterMark, objectMode, produce) {
   }
 }
 
-runTest(100, false, function() { return Buffer.allocUnsafe(100); });
-runTest(10, false, function() { return Buffer.from('xxxxxxxxxx'); });
-runTest(1, true, function() { return { foo: 'bar' }; });
+runTest(100, false, common.mustCall(function() { return Buffer.allocUnsafe(100); });
```

Comment:
There's a missing `)` at the end of this line. This is a syntax error. This file won't run. (It looks the pull request template was deleted. It asks that `make -j4 test` before submitting the pull request.)